aerogear/aerogear-android-authz
# Agdroid 319

This includes support for performing OAuth based authentication using web browser intents.

How To Test : 
 * make sure you have `aerogear-android-core', `aerogear-android-security', `aerogear-android-pipe', and `aerogear-android-store',  installed at 2.2.0-SNAPSHOT (ie latest from master).
 * fetch https://github.com/secondsun/aerogear-android-cookbook/tree/AGDROID-319 and configure the **Shoot and Share** demo
 * Make sure you have a keycloak instance running of the Shoot and Share server.
 ** Note if you are using KC 1.2.0 the realm.json file did not import correctly for me and I had to set the realm up manually
 * Take a picture and upload with keycloak providing the authentication
 * Profit

Todo and Notes : 
 * As per the ML post from last week, this works best if your service support custom URI schemas.  We need up update the documents for this
 * Once people have +1'd this I will merge and submit a PR for the ShootAndShare demo.
